engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -108,9 +105,10 @@ trait Backend {
   def engineFunctions(fileSystems: List[FileSystem], workflowContext: WorkflowContext): WorkflowEngineFunctions
 
   /**
-   * Do any work that needs to be done <b>before</b> attempting to restart a workflow.
-   */
-  def prepareForRestart(restartableWorkflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit]
+    * Assume JobKey was in a 'Running' state when the server was shut down.
+    */
+  def isResumable(key: JobKey, executionInfo: Map[String, Option[String]]): Boolean = false
+  def isRestartable(key: JobKey, executionInfo: Map[String, Option[String]]): Boolean = false
```

Comment:
The new interface only has one method for this feature no ? (called "resume") - I feel like this distinction will be lost anyway when we switch to PBE :/